src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -257,6 +257,11 @@ protected boolean shouldProcessLine(ConcurStandardAccountingExtractDetailLine sa
                 return true;
             case ConcurConstants.PAYMENT_CODE_UNIVERSITY_BILLED_OR_PAID :
                 reportCorporateCardPayment(saeLine);
+                // Temporary hack to ignore personal expense lines until their handling has been implemented.

```

Comment:
Ok, I'll remove the comment. Though it would be helpful to have, I'll be removing it almost immediately when implementing KFSPTS-8359.